recipes/codan/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.2" %}
+
+package:
+  name: codan
+  version: {{ version }}
+
+source:
+  url: https://github.com/pedronachtigall/CodAn/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 34962ac67dc94da0bff6e032568e38388726bf1d7efceed462d6b7c841b087b7
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  run:
+    - python >=3.6
+    - biopython 1.69
```

Comment:
Do you need this exact version of biopython, or will newer versions also presumably work? The difficulty with requesting an exact version is that in 2 years biopython will have newer versions out and this package will no longer be compatible with them, even if it could reasonably work.

Reply:
First, I noticed that you changed the build.sh by commenting the lines 6,7 and 8.
```
#mkdir -p $PREFIX/for_MacOS_users
#chmod +x for_MacOS_users/*
#cp for_MacOS_users/* $PREFIX/for_MacOS_users
```
We need to keep the lines uncommented. They are necessary to make CodAn work properly in macOS systems (see post-link.sh script).
Please, keep it as it was before.

About the biopython version, I will make a quick test.

Reply:
About the biopython, it can be the latest release of biopython.
It will work fine.